src/components/DropDownMenu.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+  export let label: string;
+  export let id: string;
+  export let options: Record<string, string>;
```

Comment:
curious as to why this is not an array of strings. What's your thinking here?

Reply:
other code uses:

```
export const ProviderMap: Record<string, string> = {
  Rococo: 'wss://rpc.rococo.frequency.xyz',
  Mainnet: 'wss://0.rpc.frequency.xyz',
  Localhost: 'ws://127.0.01:9944',
  Other: 'a custom endpoint',
};
```